modules/service_container_doctrine/tests/modules/service_container_doctrine_test/src/Plugin/Plugin1/Plugin1A/Plugin1A.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Drupal\service_container_doctrine_test\Plugin\Plugin1A;
```

Comment:
No this is wrong:

src/Plugin/Plugin1/Plugin1A == Drupal\service_container_doctrine_test\Plugin\Plugin1\Plugin1A